traceapp/vis.go

```diff
@@ -13,6 +14,8 @@ import (
 	_ "sourcegraph.com/sourcegraph/appdash/sqltrace"
 )
 
+var ErrTimelineItemValidation = errors.New("timeline item validation error")
```

Comment:
Does this identifier need to be exported?

If so, golint would say it needs to be documented.